sharepoint/sharepoint-ps/sharepoint-online/Register-SPODataEncryptionPolicy.md

```diff
@@ -36,7 +36,14 @@ For more information, see [Controlling your data in Office 365 using Customer Ke
 Register-SPODataEncryptionPolicy -Identity https://contoso.sharepoint.com -PrimaryKeyVaultName <string> -PrimaryKeyName <string> -PrimaryKeyVersion <KeyVersion>
 ```
 
-This example registers the DEP used with SharePoint Online and OneDrive for Business to start using the given primary key
+This example registers the DEP used with SharePoint Online and OneDrive for Business to start using the given primary key.
+
+### Example 2
+
+```powershell
+Register-SPODataEncryptionPolicy -Identity https://contoso.sharepoint.com -PrimaryKeyVaultName 'stageRG3vault' -PrimaryKeyName 'SPKey3' -PrimaryKeyVersion 'f635a23bd4a44b9996ff6aadd88d42ba' -SecondaryKeyVaultName 'stageRG5vault' -SecondaryKeyName 'SPKey5' -SecondaryKeyVersion '2b3e8f1d754f438dacdec1f0945f251a’
```

Comment:
```suggestion
Register-SPODataEncryptionPolicy -Identity https://contoso.sharepoint.com -PrimaryKeyVaultName 'PKVaultName1' -PrimaryKeyName 'PrimaryKey1' -PrimaryKeyVersion 'f635a23bd4a44b9996ff6aadd88d42ba' -SecondaryKeyVaultName 'SKVaultName1' -SecondaryKeyName 'SecondaryKey2' -SecondaryKeyVersion '2b3e8f1d754f438dacdec1f0945f251a’
```

Also, let's change the example as I see it's the same we found here:
https://www.ciscolive.com/c/dam/r/ciscolive/emea/docs/2019/pdf/BRKCOC-2384.pdf

Reply:
totally agree.